pkg/cmd/install/install.go

```diff
@@ -48,11 +50,24 @@ func installOperator(cmd *cobra.Command, args []string) error { //nolint[:unpara
 	if err != nil {
 		return err
 	}
+
 	local, err := cmd.Flags().GetBool("local")
 	if err != nil {
 		return err
 	}
 
+	if local && !cmd.Flags().Changed("namespace") {
+		kubeCfg := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+			clientcmd.NewDefaultClientConfigLoadingRules(),
+			&clientcmd.ConfigOverrides{},
+		)
+
+		if namespace, _, err = kubeCfg.Namespace(); err != nil {
+			log.Error(err, "failed to read k8s config")
```

Comment:
Maybe make the error something like: Failed to read kubeconf 'config file path' to get the current namespace, please provide a namespace manually via -namespace?

Reply:
Oh, I missed that one. Providing PR in a minute. #314